packages/arb-token-bridge-ui/src/util/TokenTransferDisabledUtils.ts

```diff
@@ -18,15 +20,33 @@ const transferDisabledTokens: { [chainId: number]: TransferDisabledToken[] } = {
       symbol: 'FU',
       l1Address: '0x43df01681966d5339702e96ef039e481b9da20c1',
       l2Address: '0x9aee3C99934C88832399D6C6E08ad802112eBEab'
-    },
-    {
-      symbol: 'USDT', // disable USDT transfers during OFT migration
-      l1Address: CommonAddress.Ethereum.USDT,
-      l2Address: CommonAddress.ArbitrumOne.USDT
     }
   ]
 }
 
+export const isUsdtTransferBetweenEthAndArbOne = ({
```

Comment:
Making this common function exportable since we need it in 2 places.
1. To disable the transfer in `useTransferReadiness`
2. To show custom content in the Transfer Disabled Dialog.